openmc/deplete/results_list.py

```diff
@@ -296,3 +298,72 @@ def get_step_where(
             "relative tolerances {} and {}.".format(
                 time, time_units, atol, rtol)
         )
+
+    def export_to_materials(self, burnup_index, nuc_with_data=None):
+        """Return openmc.Materials created from these depletion results.
+
+        Parameters
+        ----------
+        burn_index : int
+            Index of burnup step to evaluate
```

Comment:
It would be good to have a "See Also" section that mentions the `get_step_where` method

Reply:
Good idea! I've now added this.